sdk/storage/storage-blob/CHANGELOG.md

```diff
@@ -2,12 +2,14 @@
 
 ## 12.1.1 (Unreleased)
 
+- Bug fix - Blob SAS's `sr` field is now properly set when generating SAS for a blob using a stored policy with `signedpermissions`.
```

Comment:
Where does the term `signedpermissions` come from?

Reply:
[https://docs.microsoft.com/en-us/rest/api/storageservices/create-service-sas](https://docs.microsoft.com/en-us/rest/api/storageservices/create-service-sas), let me add link for introduction propose.

---
In reply to: [380162274](https://github.com/Azure/azure-sdk-for-js/pull/7397#discussion_r380162274) [](ancestors = 380162274)